.Xresources

```diff
@@ -57,6 +57,15 @@ st.boxdraw_bold: 0
 ! braille (U28XX):  1: render as adjacent "pixels",  0: use font
 st.boxdraw_braille: 0
 
+! opacity==255 means what terminal will be not transparent, 0 - fully transparent
+! (float values in range 0 to 1.0 could be used instead)
+st.opacity:      255
+! (0|1) don't pre-multiply color values of the pixels when using transparency
+! (useful for the systems where composite manager not running all the time,
+! as color pre-multiplication will make the "opaque" colors look darker/lighter
+! without the composition)
+st.disableAlphaCorrection: 0
```

Comment:
also it's a bit bothering me what in xresources we have keynames like `st.foobar`, `st.foo_bar` and `st.fooBar`

Reply:
sameee. I think disableAlphaCorrection should maybe be changed to snake_case, and then maybe we consider moving the example .Xresources file to the wiki

Reply:
now i'm on my pc, pushed the change here